desktop/renderer/index.desktop.js

```diff
@@ -8,6 +8,11 @@ import Nav from '../../react-native/react/nav'
 import injectTapEventPlugin from 'react-tap-event-plugin'
 import {isDev} from '../../react-native/react/constants/platform'
 
+// For Remote Components
+import remote from 'remote'
+const ipc = remote.require('ipc')
+// import ipc from 'ipc'
```

Comment:
Cool.  As a heads up, for me s/ipc/ipcMain/g worked great, but s/ipc/ipcRenderer/g in my component gave me a weird crash, haven't figured it out why yet.